GLMakie/src/events.jl

```diff
@@ -99,7 +99,9 @@ function Makie.mouse_buttons(scene::Scene, window::GLFW.Window)
     event = scene.events.mousebutton
     function mousebuttons(window, button, action, mods)
         @print_error begin
-            event[] = MouseButtonEvent(Mouse.Button(Int(button)), Mouse.Action(Int(action)))
+            if Int(button) < 3  # ignore mouse buttons beyond 3
```

Comment:
Should we ignore them, or just pass through?

Reply:
I got an error in makielayout/mousestatemachine.jl.

To allow extra buttons to be used, the enum `MouseEventTypes` would need to be expanded hugely, and all the `@match` uses in that file too. I'd be willing to go through and do that if it's of interest.

Reply:
Ok, I think we should merge this PR first then, and fix this once demand arises!

Reply:
Sounds good. I have a mouse with lots of buttons on the side and I'm almost always annoyed when they actually do something!

Reply:
I think rather than discarding the event we should ignore it in mousestatemachine then.